contrib_bots/lib/translate.py

```diff
@@ -0,0 +1,86 @@
+# See zulip/contrib_bots/lib/readme.md for instructions on running this code.
+from __future__ import print_function
+import sys
+import logging
+try:
+    from PyDictionary import PyDictionary as dictionary
+except ImportError:
+    logging.error("Dependency Missing!")
+    sys.exit(0)
+
+#Uses Python's Dictionary module
+#  pip install PyDictionary
+
+def get_translate_result(original_content):
+    if len(original_content.split()) < 2:
+        return "Not enough cases"
+    elif len(original_content.split()) > 3:
+        return "Too many cases"
+    else:
+        main_content = original_content.strip().split(' ', 1)[1]
```

Comment:
You want main_content to be something like `word code` right?
`main_content = original_content.strip().split(' ')[1:]`
[1] just get you the word